app/classifier/tutorial.cjsx

```diff
@@ -104,7 +104,7 @@ module.exports = createReactClass
     isIE = 'ActiveXObject' of window
     if isIE
       tutorialStyle = height: '85vh'
-    <StepThrough ref="stepThrough" className="tutorial-steps" style={tutorialStyle}>
+    <StepThrough ref={(node) => @stepThrough = node} className="tutorial-steps" style={tutorialStyle}>
```

Comment:
Ditto about variable naming here. `@stepThrough` is a reference to a React component.